specifyweb/frontend/js_src/lib/components/WbUtils/index.tsx

```diff
@@ -14,13 +14,15 @@ import type { WbUtils } from './Utils';
 
 export function WbUtilsComponent({
```

Comment:
some raw ideas for how to reduce the visual size of this component:
- have a small button icon to collapse the "search", "replace", "search settings" and "search results" elements
- replace the navigation button labels with icons, with title being shown in a tooltip on hover